tasks/47_string-expansion/string-expansion.js

```diff
@@ -0,0 +1,20 @@
+function stringExpansion(str) {
+  const regLetter = new RegExp(/^[A-Za-z]+$/);
+  const regNumber = new RegExp(/^[0-9]$/);
+  if (regLetter.test(str)) {
+    return str;
+  }
+  let newStr = '';
+  for (let i = 0; i < str.length; i++) {
+    if (regNumber.test(str[i])) {
+      let j = 1;
```

Comment:
What is J ?

Could you please add JSDoc for this functions

Reply:
I gave the variable a more understandable name